tools/MultiProjectValidator/project.json

```diff
@@ -5,7 +5,7 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "Microsoft.NETCore.App": "1.0.0-rc3-*",
+    "Microsoft.NETCore.App": "1.0.0-rc2-*",
```

Comment:
It should be using up-to-date dependencies.  At some point in the future we will want to take a new CoreFX build, and then the tool will break because the new CoreFX isn't compatible with the "rc2" build of NETCore.App.

Either we:

1. Make it not use NETCore.App (like our build scripts do - using NETStandard.Library and NETCore.Runtime)
1. Make it consume the NETCore.App from our build (i.e. you need to change the `dotnet restore` to add `-f Dirs.CoreHost -f Dirs.CoreHostStubPackages`